kubernetes_deploy/production/ingress.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion:  networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: laa-fala
+  name: laa-fala-v122
   namespace: laa-fala-production
   annotations:
     external-dns.alpha.kubernetes.io/set-identifier: laa-fala-v122-laa-fala-production-green
```

Comment:
this still needs to change so should be `laa-fala-laa-fala-v122-production-green`

Reply:
I think it's <ingress-name>-<namespace>-<colour>
So the ingress name is laa-fala-v122